frontend/package.json

```
@@ -5,7 +5,8 @@
   "main": "./src/main.js",
   "scripts": {
     "serve": "vite",
-    "build": "vite build"
+    "build": "vite build",
```

```suggestion
    "build": "vite build && npm run copy",
```
Sorry forgot to add we should do this.

---

@lauratsui and @thchia 
you can add options to our existing vite.config.js
just install rollup copy plugin

```javascript
import { createVuePlugin } from 'vite-plugin-vue2'

export default {
  rollupInputOptions: {
    plugins: [
      /// Rollup copy plugin
    ]
  },
  alias: {
    'vue': 'vue/dist/vue.esm.js'
  },
  plugins: [
    createVuePlugin()
  ]
}
```

---

@aelbore I tried this... but it didn't seem to work. Build was still ok, but the copying did not happen. Can you verify?

---

npm install aria-fs --save-dev
import { createVuePlugin } from 'vite-plugin-vue2'
import { copyFiles } from 'aria-fs'

export default {
  rollupInputOptions: {
    plugins: [
      {
        name: 'copy-file',
        'buildEnd': () => copyFiles('./dist/**/*', 'build')
      }
    ]
  },
  alias: {
    'vue': 'vue/dist/vue.esm.js'
  },
  plugins: [
    createVuePlugin()
  ]
}

---

Yeah I was using our existing vite config file, but I was trying to use the rollup copy plugin directly. @lauratsui you can have a go at implementing the suggestion here instead of having the extra script in the `package.json`.

---

Thanks @aelbore. @thchia, @szktm and I tried it out but there were a few errors and we're still not entirely sure what's going on. We think we can proceed to merge with the original version first to get the copying up and running in the deployment first, and we'll come back to this at a later point.

---

The build is working, just that the copy script appears to have no effect. Also, we had to use `writeBundle` instead of `buildEnd`, as at `buildEnd`, the `dist` folder is not created yet.